### PR TITLE
Integrate as part of IAccountConfiguration Fireblocks and Dfns

### PR DESCRIPTION
**Pull request by marcosio**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by MiguelLZPF** [comment]
After wizard tests there are a few fixes pending:

- [ ] Private key loop
- [ ] Account ID asked 2 times
- [ ] Public key question

I'm on it, changing PR state to draft

File: cli/hsca-config.sample.yaml [hunk 1]
```diff
@@ -35,6 +33,27 @@ accounts:
...
+        apiSecretKey: <Fireblocks API Secret Key>
+        apiKey: <Fireblocks API Key>
+        baseUrl: <Fireblocks API Base URL>
+        assetId: <Fireblocks Asset Id>
+        vaultAccountId: <Fireblocks Vault Account Id>
+        hederaAccountPublicKey: <Fireblocks Hedera Account Public Key>
+      dfns:
+        authorizationToken: <DFNS Authorization Token>
+        credentialId: <DFNS Credential Id>
+        privateKey: <DFNS Private Key>
```
Comment (M-Francia) [inline cli/hsca-config.sample.yaml:51]: public key?

File: cli/src/app/service/configuration/SetConfigurationService.ts [hunk 2]
```diff
@@ -378,54 +406,232 @@ export default class SetConfigurationService extends Service {
...
+    return privateKey;
+  }
+
+  private async askForDfnsAppId(): Promise<string> {
+    let appId = '';
+    const appIdRegExpValidator = /ap-[a-z0-9]+-[a-z0-9]+-[a-z0-9]+/g;
+    while (!appIdRegExpValidator.test(appId)) {
+      appId = await utilsService.defaultSingleAsk(
+        language.getText('configuration.dfns.askAppId'),
+        'ap-2ng9jv-80cfc-983pop0iauf2sv8r',
```
Comment (M-Francia) [inline cli/src/app/service/configuration/SetConfigurationService.ts:617]: Should we delete the default values?

File: cli/src/app/service/configuration/SetConfigurationService.ts [hunk 3]
```diff
@@ -214,13 +224,31 @@ export default class SetConfigurationService extends Service {
...
+        case AccountType.Fireblocks:
+          accountConfig.nonCustodial = {
+            fireblocks: await this.askForFireblocksOfAccount(),
+          };
+          break;
+        default:
+          accountConfig.nonCustodial = {
+            dfns: await this.askForDfnsOfAccount(),
+          };
+          break;
```
[the remark this answers is not in the document]
Comment (marcosio) [reply cli/src/app/service/configuration/SetConfigurationService.ts:249]: Very good question. AccountType is an enum with only 3 values. 
As it is not possible to use other values than those 3, I decided to leave the DFNS resolution as a default option.

Does typeScript use out-of-enum values on it??

File: cli/src/app/service/configuration/SetConfigurationService.ts (hunk 2 above)
Comment (marcosio) [reply cli/src/app/service/configuration/SetConfigurationService.ts:617]: I set it to have a better user experience. I will modify it a bit maintaining the format.

File: cli/src/app/service/configuration/SetConfigurationService.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (MiguelLZPF) [reply cli/src/app/service/configuration/SetConfigurationService.ts:249]: If you ask me I would set the default to SELF-CUSTODIAL if any, else remove the default block or ask again the same question

**Review by MiguelLZPF — APPROVED** [review]

**Review by M-Francia — APPROVED** [review]
olé

**Pull request merged**